irohad/synchronizer/synchronizer.hpp

```diff
@@ -24,8 +24,11 @@
 #include "synchronizer/synchronizer_common.hpp"
 
 namespace iroha {
-  namespace synchronizer {
+  namespace network {
+    struct Commit;
```

Comment:
Seems bit confusing that `struct` was used.
I know that `struct` and `class` are totally the same in this case. But other code typically uses `class` for the forward decl.

Reply:
Since commit is a struct, we have to use struct in forward declaration.